pghoard/webserver.py

```diff
@@ -37,6 +38,11 @@ class OwnHTTPServer(PoolMixIn, HTTPServer):
     pool = ThreadPoolExecutor(max_workers=10)
     requested_basebackup_sites = None
 
+    def __init__(self, server_address, RequestHandlerClass):
+        if "::" in server_address:
+            self.address_family = socket.AF_INET6
```

Comment:
Using `ipaddress.ip_address` and checking if it's a `IPv6Address` would probably more appropriate

Reply:
:+1:

Reply:
@packi the docs say that if you specify an empty http_address it should listen on all interfaces.   Of course it only listens on all IPV4 interfaces at the moment.

As part of this change, Im assuming I should not change the behavior to listen on all IPV6 and IPV4 addresses?

Reply:
I would leave it as is for now.